packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -372,6 +372,11 @@ class _ScreenshotData {
   set screenshotOffset(Offset offset) {
     containerLayer.offset = offset;
   }
+
+  /// Releases allocated resources.
```

Comment:
Can you add @mustCallSuper?

Reply:
Thanks for the review. I added @mustCallSuper in the latest commit.